src/shared/api/schemas.ts

```diff
@@ -31,6 +31,7 @@ const PaginationParams = z
 export type PaginationParams = z.infer<typeof PaginationParams>;
 
 const Metadata = z.record(z.string(), z.any());
+const TransactionStatus = z.enum(['Committed', 'Rejected']);
```

Comment:
```suggestion
const TransactionStatus = z.enum(['Committed', 'Rejected']);
type TransactionStatus = z.infer<typeof TransactionStatus>
```

It is common type used in other places too.